lib/matplotlib/backend_bases.py

```diff
@@ -820,7 +820,7 @@ def get_antialiased(self):
 
     def get_capstyle(self):
         """
-        Return the capstyle as a string in ('butt', 'round', 'projecting').
+        Return the default `.CapStyle`.
         """
```

Comment:
If the summary line is short enough, we put it on one line, including the quotes:

~~~
    def get_capstyle(self):
        """Return the default `.CapStyle`."""
~~~